src/hoi4_world/countries/hoi4_countries_converter.cpp

```diff
@@ -17,7 +17,188 @@ std::map<std::string, hoi4::Country> hoi4::ConvertCountries(const std::map<int,
+           }},
+       {"law_wealth_voting",
+           {
+               {"democratic", 25},
+               {"communism", -50},
+               {"fascism", 0},
+               {"neutrality", 50},
+           }},
+       {"law_census_voting",
+           {
+               {"democratic", 50},
+               {"communism", -50},
+               {"fascism", 25},
+               {"neutrality", 50},
+           }},
+       {"law_universal_suffrage",
+           {
+               {"democratic", 100},
+               {"communism", 100},
+               {"fascism", -25},
+               {"neutrality", 25},
+           }},
+       {"law_anarchy",
+           {
+               {"democratic", -10'000},
+               {"communism", 10'000},
+               {"fascism", -10'000},
+               {"neutrality", -10'000},
+           }},
+       // citizenship
+       {"law_ethnostate",
+           {
+               {"democratic", -50},
+               {"communism", -100},
+               {"fascism", 100},
+               {"neutrality", 25},
+           }},
+       {"law_national_supremacy",
+           {
+               {"democratic", -25},
+               {"communism", 0},
+               {"fascism", 75},
+               {"neutrality", 50},
+           }},
+       {"law_racial_segregation",
+           {
+               {"democratic", 25},
+               {"communism", -25},
+               {"fascism", 50},
+               {"neutrality", 50},
+           }},
+       {"law_cultural_exclusion",
+           {
+               {"democratic", 50},
+               {"communism", 0},
+               {"fascism", 25},
+               {"neutrality", 50},
+           }},
+       {"law_multicultural",
+           {
+               {"democratic", 75},
+               {"communism", 75},
+               {"fascism", -50},
+               {"neutrality", 0},
+           }},
+       // church and state
+       {"law_state_religion",
+           {
+               {"democratic", 0},
+               {"communism", -75},
+               {"fascism", 50},
+               {"neutrality", 75},
+           }},
+       {"law_freedom_of_conscience",
+           {
+               {"democratic", 25},
+               {"communism", -50},
+               {"fascism", 0},
+               {"neutrality", 50},
+           }},
+       {"law_total_separation",
+           {
+               {"democratic", 50},
+               {"communism", 50},
+               {"fascism", 0},
+               {"neutrality", 0},
+           }},
+       // bureaucracy
+       {"law_hereditary_bureaucrats",
+           {
+               {"democratic", -25},
+               {"communism", -75},
+               {"fascism", -25},
+               {"neutrality", 50},
+           }},
+       {"law_appointed_bureaucrats",
+           {
+               {"democratic", 25},
+               {"communism", 50},
+               {"fascism", 50},
+               {"neutrality", 25},
+           }},
+       {"law_elected_bureaucrats",
```

Comment:
Elected is heavy for communist/trade unions. Reverse with appointed.

... But generally these values need a hefty balancing/tweaking pass, once you split them out into a config file.

Reply:
These are pulled from the proposed configs in the Discord, and I'll be leaving them for others to balance while I'm on vacation.